lib/active_model/serializer.rb

```diff
@@ -9,8 +9,10 @@ class Serializer
     autoload :Adapter
     autoload :Lint
     autoload :Associations
+    autoload :KeyFormat
```

Comment:
Here you are including the file when it's evaluated, there is no advantage on the autoload. It's not a blocking issue, but in this case you could use the usual `require` or `require_relative` without the autoload. 
We are already updating it to do the same thing with the other included files.